tasks/getHistoricalData.ts

```diff
@@ -85,6 +85,12 @@ task("getHistoricalData", "Retrieves the historical rates")
   )
   .addFlag("compound", "Get rates data from Compound")
   .addFlag("aave", "Get rates data from Aave")
+  .addOptionalParam(
+    "rate",
```

Comment:
Poor name for param.

Would a --borrow flag be more appropriate, with lending used if flag not set?